src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -127,6 +127,8 @@ public static void AddDefaultServices(this IServiceCollection services, GlobalSe
         {
             // Required for UserService
             services.AddWebAuthn(globalSettings);
+            // Required for HTTP calls
+            services.AddHttpClient();
```

Comment:
`service.AddHttpClient()` lets us [configure the `HttpClient`](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-6.0#named-clients) right here with a given name. This would save us from having to pass in the globalSettings to get the baseUri and what not and wouldn't require every caller to add in their own `Accept` header. I think it's something we should think about and at the very least put something on the tech dept board.

Reply:
The only reason I didn't do this is because `BaseIdentityClientService` allows different Uris to be passed in for the Client and Identity `HttpClients`. I think you're right though, it would be nice to have in one spot and definitely something for the tech debt board.